src/api/XRSessionWithLayer.ts

```diff
@@ -288,7 +288,9 @@ export class XRSessionWithLayer {
 	public updateRenderState(XRRenderStateInit: XRRenderStateWithLayer): void {
 		this.existingBaseLayer = XRRenderStateInit.baseLayer
 
-		this.layers = XRRenderStateInit.layers || []
+		if (XRRenderStateInit.layers){
```

Comment:
nit: Can you run `npm run format` on the pull request before merging?

Reply:
done